### ISSUE TITLE
Menu applet unable to configure.    

### ISSUE THREAD

**Issue by ChooChooAl** [linuxmint/mint-19.1-beta#7]
**Describe the bug**
Right clicking the menu icon, typically bottom left in panel, and then clicking 'configure' does nothing

**Expected behavior**
A window should appear to edit parts of the menu.

**Frequency**
Does the issue happen:
[X ] Always
[ ] Randomly

**Edition (Desktop environment):**
In which edition is this happening?
[X ] Cinnamon Edition
[ ] MATE Edition
[ ] Xfce Edition

**Live or post-install:**
Indicate if this is happening after the installation, or during the live session:
[X ] Post-install
[ ] Live session

**Locale:**
LANG=en_US.UTF-8

**Additional context**
Not sure this is part of a larger issue but the slideshow controls applet does not show any icon.  It's there and I can right click the blank space where the icon is and configure etc but just no icon.

**Comment by clefebvre** [comment]
The slideshow applet is fixed in 4.0.5.

**Comment by NikoKrause** [comment]
Try the following command:
```

**Comment by ChooChooAl** [comment]
In the terminal running the above command returns:
    Segmentation fault (core dumped)
I did it for weather@mockturtl and I see what it's supposed to do, but menu@cinnamon.org dumps.
Cool trick.

**Comment by ChooChooAl** [comment]
I think it's modern, the larger panel.  (Kinda like it too). 
I did just rebooted the live CD in VirtualBox and just using the live session, config menu works.

Finally found the issue.   It seems to be my icon theme I built.  It's been working great for the past 3 versions and even works in 19.0 V2, but not in 19.1 Beta.   Something has changed.  Not sure what but no matter Modern or Traditional desktop, it's still all comes down to the icon theme.   And my theme breaks the right click config on menu@cinnamon.org.  

Would be nice to know what does it.   An icon theme isn't that hard to make considering I just replaced a few icons with my own and added a dozen or 2 more.

**Comment by NikoKrause** [comment]
Would be nice to know, which icon theme breaks the menu settings. Can you tell us, which icon theme you're using?

**Comment by ChooChooAl** [comment]
It's a custom icon theme.  I derived it from Oxygen I think.   Just spent the past 2 years slowly changing a few icons to fit my whim.    I have discovered in the base folder there are the typical 8x8 16x16 etc folders and then one scalable folder.   Inside the scalable is devices, places, actions.      Removing the scalable folder fixes this issue.  I'm now playing with different options.   The theme index said all scalable were 16x16 but I found about a dozen icons that were not, so I fixed that but it did no good other than make things technically right.   
I'm currently out of play time but I want to fix this.   The scalable folder had all the nice icons for the battery and volume applets etc that were colorized.  It was just a nice tweak.   I'll plug away.   
I did notice the new oxygen theme has a lot of broken icon links and the index says base/16x16 etc but then also says 16x16 (without base).   Not sure if that's design or not since it inherits other themes.  
If I find anything I'll note it here later.

**Comment by ChooChooAl** [comment]
Final update.  I'm still not sure what was wrong but I used the gnome icon theme as a base and just purged all the icons and replaced with my own.  Had to build a new 24x24 set but that was simple.  Even the scalable folder icons work.  I'm not 100% sure of all the code in the index file so there may have been some glitch there.    
I do have to note that when using the Oxygen theme, there were a ton of broken links and my bulk copy with merge folders and overwrite icons failed miserably.   With the Oxygen theme  I purged all the files leaving just the folders and then copied my icons into the folders respectively.    Even still I have the same folder structure and files, just a different index file.   Bottom line is all the issues were my own fault.  (That or the new mint is less tolerant of index syntax errors).
